caffe2/core/nomnigraph/include/nomnigraph/Generated/OpClasses.h

```diff
@@ -475,6 +475,62 @@ class Receive : public NeuralNetOperator {
   string Source;
 };
 
+class BatchNormalization : public NeuralNetOperator {
```

Comment:
One general question is do we need to create a corresponding class on nomnigraph for c2 operator or we can just rely on the `OperatorDef*` attribute?

Reply:
It isn't strictly necessary to use nomnigraph, but the transformations need to know about that and use string comparison instead of isa<ClassName>() style checks (which are substantially more efficient).

I've tried to make it as seamless as possible with ops.def syntax for adding new classes, though

Reply:
Fair enough.